newman/dev/rate-spread/rate-spread-test.json

```diff
@@ -0,0 +1,165 @@
+        "value": 0,
+        "type": "int"
+      },
+      {
+        "key": "requestBody",
+        "value": "",
+        "type": "string"
+      },
+      {
+        "key": "expectedResponse",
+        "value": "",
+        "type": "string"
+      },
+      {
+        "key": "sourceFile",
+        "value": "",
+        "type": "string"
+      },
+      {
+        "key": "currentDate",
+        "value": "",
+        "type": "string"
+      }
+    ],
+    "event": [
+      {
+        "listen": "prerequest",
+        "script": {
+          "type": "text/javascript",
+          "exec": [
+            "testNumber = pm.variables.get(\"testNumber\");",
+            "let testData = pm.iterationData.get(\"testData\")",
+            "let requestBody = JSON.stringify(testData[testNumber]? testData[testNumber].requestBody:{} )",
+            "let expectedResponse = JSON.stringify(testData[testNumber]? testData[testNumber].expectedResponse:{} )",
+            "let sourceFile = testData[testNumber].sourceFile",
+            "let currentDate = new Date().toISOString().slice(0,10)",
+            "++testNumber",
+            "pm.variables.set(\"testNumber\",testNumber);",
+            "pm.variables.set(\"requestBody\",requestBody);",
+            "pm.variables.set(\"currentDate\",currentDate);",
+            "pm.variables.set(\"sourceFile\",sourceFile);"
+
+          ]
+        }
+      }
+    ],
+    "item": [
+      {
+        "name": "Single Ratespread Check",
+        "event": [
+          {
+            "listen": "test",
+            "script": {
+              "type": "text/javascript",
+              "exec": [
+                "pm.test(\"Status code is 200\",function () {",
+                "      pm.response.to.have.status(200);",
+                "});",
+                "",
+                "pm.test(\"Recieved a Rate Spread\", function () {",
+                "let actualResponse = pm.response.json()",
+                "let testData = pm.iterationData.get(\"testData\")",
+                "let testDataChunk = testData[testNumber-1]",
+				"console.log(JSON.stringify(actualResponse))",
+				"pm.expect(actualResponse).to.be.eql(testDataChunk.expectedResponse);",
+
+              "});"            ]
+            }
+          }
+        ],
+        "request": {
+          "url":  "{{rate-spread}}",
+          "method": "POST",
+          "header": [
+            {
+              "key": "Content-Type",
+              "value": "application/json"
+            }
+          ],
+          "body": {
+            "mode": "raw",
+            "raw": "{{requestBody}}"
+        },
+          "description": "Single Rate Spread Check"
+        },
+        "response": []
+      },
+      {
+        "name": "Current Date Ratespread Check",
+        "event": [
+          {
+            "listen": "test",
+            "script": {
+              "type": "text/javascript",
+              "exec": [
+                "pm.test(\"Status code is 200\",function () {",
```

Comment:
Since we can't predict what the new values are going to be until the APOR updates is is OK to confirm the path works.